api/main_endpoints/routes/User.js

```diff
@@ -250,6 +251,47 @@ router.get('/callback', async function(req, res) {
     });
 });
 
+// /POST 'updatePagesPrintedFromDiscord'
+// check api key
+// req.body.discordID req.body.printedPages
+// User.edit({printedPages}, (err, result))
+// write api tests for this api endpoint as well
+// api function in discord repo, update command file
+// test
+
```

Comment:
delete these random comments, and instead if you want to, you can just explain how it works in multi-line comments